dev_cmds/setup-dev.ts

```diff
@@ -0,0 +1,52 @@
+import { execPiped, runAsyncMain } from "devcmd";
+import { YARN_COMMAND } from "./utils/commands";
+import { devcmdPackageDir, devPath, singlePackageJsonExampleDir } from "./utils/paths";
+import fs from "fs-extra";
+
+async function main() {
+
+  if(await fs.pathExists(devPath)) {
```

Comment:
there's some inconsistent formatting here, please make sure to run prettier and update this